engine/graphics/proto/graphics/graphics_ddf.proto

```diff
@@ -171,6 +180,7 @@ message TextureImage
     // When count > 1 count mipmaps are laid out contiguously in memory and
     // the mip_map_offset should reflect that, e.g. times 6 for cubemaps
     required uint32 count       = 3;
+    optional uint32 usage_flags = 4 [default = 1];
```

Comment:
is this a good name? or should it be called something else? "flags", "hints", "create_flags" or something else?

Reply:
I think it's pretty good!